src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -161,6 +161,12 @@ Network port to use when connecting to an Android device with [ADB (Android Devi
 
 Environment variable: `$WEB_EXT_ADB_PORT`
 
+#### \--adb-remove-old-artifacts
+
+Removes automatically any old artifacts dir left on the target adb device by a previous `web-ext run` command.
```

Comment:
I'd like to offer a bit more explanation on the context, so that users know when it's needed.

How about:

> \`web-ext run\` should remove temporary files that were written to target device. This may fail, for example when the device is disconnected before \`web-ext\` exited. This flag forces \`web-ext\` to look for such files and remove them if discovered.

Reply:
@Rob--W I tweaked the description a bit, it is slightly different from the version you have proposed (e.g. `web-ext run` does actually automatically detect if there are old artifacts left on the device by default, but it doesn't remove them unless the user has explicitly asked it with this flag). 

Let me know how the new versions looks to you.